dlls/Makefile.mingw

```diff
@@ -0,0 +1,206 @@
+#
+# Half-Life Full SDK 2.3 hl_i386.so Makefile for x86 Linux
+#
+# October 2002 by Leon Hartwig (hartwig@valvesoftware.com)
+#
+
+DLLNAME=hl
+
+ARCH=i386
+
+#make sure this is the correct compiler for your system
+CC=gcc
+CXX=g++
+
+DLL_SRCDIR=.
+ENGINE_SRCDIR=../engine
+COMMON_SRCDIR=../common
+WPN_SHARED_SRCDIR=./wpn_shared
+PM_SHARED_SRCDIR=../pm_shared
+GAME_SHARED_SRCDIR=../game_shared
+
+DLL_OBJDIR=$(DLL_SRCDIR)/obj
+WPN_SHARED_OBJDIR=$(WPN_SHARED_SRCDIR)/obj
+PM_SHARED_OBJDIR=$(PM_SHARED_SRCDIR)/obj
+GAME_SHARED_OBJDIR=$(GAME_SHARED_SRCDIR)/obj
+
+#BASE_CFLAGS=	-Dstricmp=strcasecmp -D_strnicmp=strncasecmp -Dstrnicmp=strncasecmp \
+		-DCLIENT_WEAPONS -DNO_VOICEGAMEMGR
+
+BASE_CFLAGS= -DCLIENT_WEAPONS -Dstricmp=strcasecmp -Dstrnicmp=strncasecmp -D_snprintf=snprintf \
+	-fno-exceptions -DNO_VOICEGAMEMGR -Wno-conversion-null -Wno-write-strings -std=gnu++98 -Wno-invalid-offsetof
+
+
+#safe optimization
+CFLAGS=$(BASE_CFLAGS) -m32 -w -O1
+
+#full optimization
+#CFLAGS=$(BASE_CFLAGS) -w -O1 -m486 -ffast-math -funroll-loops \
```

Comment:
>full optimization

>-O1 -m486

LOL.

Reply:
-ffast-math is better =_=

Reply:
Just copypasted from dlls/Makefile

Reply:
Ah... Ok.

Reply:
Unified Makefile would be better anyway.